packages/bootstrap-4/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -2,48 +2,46 @@ import React from "react";
 
 import { FieldTemplateProps } from "@rjsf/core";
 
-import FormControl from "@material-ui/core/FormControl";
-import FormHelperText from "@material-ui/core/FormHelperText";
-import List from "@material-ui/core/List";
-import ListItem from "@material-ui/core/ListItem";
-
 import Form from "react-bootstrap/Form";
+import ListGroup from "react-bootstrap/ListGroup";
 
 const FieldTemplate = ({
   id,
   children,
   displayLabel,
-  required,
   rawErrors = [],
   rawHelp,
   rawDescription,
 }: FieldTemplateProps) => {
   return (
-    <FormControl
-      fullWidth={true}
-      error={rawErrors.length ? true : false}
-      required={required}>
+    <Form.Group>
       {children}
       {displayLabel && rawDescription ? (
-        <Form.Text>{rawDescription}</Form.Text>
+        <Form.Text className={rawErrors.length > 0 ? "text-danger" : ""}>
+          {rawDescription}
+        </Form.Text>
       ) : null}
       {rawErrors.length > 0 && (
-        <List dense={true} disablePadding={true}>
+        <ListGroup as="ul">
           {rawErrors.map((error, i: number) => {
             return (
-              <ListItem key={i} disableGutters={true}>
-                <FormHelperText id={id}>{error}</FormHelperText>
-              </ListItem>
+              <ListGroup.Item as="li" key={i} variant="danger">
```

Comment:
We shouldn't use indexes as keys. Here's why: https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c

I'd rather use `error` as key in this case, assuming it's a plain string?